src/ansys/pre_commit_hooks/add_license_headers.py

```diff
@@ -252,49 +174,63 @@ def find_files_missing_header():
     parser = argparse.ArgumentParser()
     args = set_lint_args(parser)
 
-    # Get custom specified directories, copyright, template, and/or license
-    dirs = args.loc.split(",")
+    # Get committed files, the year, and custom copyright, template, and/or license arguments
+    files = args.files
     copyright = args.custom_copyright
     template = args.custom_template
     license = args.custom_license
-    changed_headers = False
-
-    # Get current year for license file
+    changed_headers = 0
     year = dt.today().year
 
+    # Get root directory of the git repository.
+    git_repo = git.Repo(os.getcwd(), search_parent_directories=True)
+    git_root = git_repo.git.rev_parse("--show-toplevel")
+
+    # Update file paths to be absolute paths with correct separators
+    for i in range(0, len(files)):
```

Comment:
Why don't you do ``for file in files``?

Reply:
It's always better to handle it that way rather than with ``range``

Reply:
I used range instead because I wanted update each file in the ``files`` list with the file's absolute path. I don't think I would be able to do that with ``for file in files``

Reply:
I see, yeah. Okay let's leave it as is.